Typescript check error

Typescript check falling with  error:

```
❯ yarn tsc
yarn run v1.22.22
$ /home/alex/Documents/work-projects/tests/aidbox-react/node_modules/.bin/tsc
src/hooks/service.ts:28:28 - error TS18046: 'err' is of type 'unknown'.

28             return failure(err.response ? err.response.data : err.message);
                              ~~~
src/hooks/service.ts:28:43 - error TS18046: 'err' is of type 'unknown'.

28             return failure(err.response ? err.response.data : err.message);
                                             ~~~
src/hooks/service.ts:28:63 - error TS18046: 'err' is of type 'unknown'.

28             return failure(err.response ? err.response.data : err.message);
                                                                 ~~~

Found 3 errors in the same file, starting at: src/hooks/service.ts:28

```

--- a/src/hooks/service.ts
+++ b/src/hooks/service.ts
@@ -24,7 +24,7 @@ export function useService<S = any, F = any>(
     const load = useCallback(async () => {
         try {
             return await asyncFunction();
-        } catch (err) {
+        } catch (err: any) {
             return failure(err.response ? err.response.data : err.message);
         }
         // eslint-disable-next-line react-hooks/exhaustive-deps

